src/components/header/Header.tsx

```diff
@@ -1,6 +1,184 @@
+            <div className="header-logo">
+              <Link to="/">
+                <img
+                  src={logo}
+                  alt="Our logo"
+                  className={color ? 'scroll-fill logo-image img-responsive ' : 'logo-image img-responsive'}
+                />
+              </Link>
+            </div>
+          </div>
+          <div className="header-navbar">
+            <ul className="navbar-menu">
+              <li className="menu-item">
+                <Link to="/" className={linkClassName}>
+                  Home
+                </Link>
+              </li>
+              <li className="menu-item">
+                <Link to="/shop" className={linkClassName}>
+                  Shop
+                </Link>
+              </li>
+              <li className="menu-item">
+                <Link to="/contact" className={linkClassName}>
+                  Contact
+                </Link>
+              </li>
+              <li className="menu-item pages-menu">
+                <Link to="/" className={linkClassName}>
+                  Pages
+                </Link>
+                <ul className="sub-menu">
+                  <li className="sub-item">
+                    <Link to="/checkout" className="menu-item-href">
+                      Checkout
+                    </Link>
+                  </li>
+                  <li className="sub-item">
+                    <Link to="/ordertracking" className="menu-item-href">
+                      Order Tracking
+                    </Link>
+                  </li>
+                  <li className="sub-item">
+                    <Link to="/wishlist" className="menu-item-href">
+                      Wishlist
+                    </Link>
+                  </li>
+                  <li className="sub-item">
+                    <Link to="/cart" className="menu-item-href">
+                      Cart
+                    </Link>
+                  </li>
+                </ul>
+              </li>
+              <li className="menu-item">
+                <Link to="/about" className={linkClassName}>
+                  About
+                </Link>
+              </li>
+            </ul>
+          </div>
+          <div className="header-icons">
+            <button className="icons-element icons-search search-btn">
+              <img src={icon1} alt="Search" className={linkIconClassName} />
+            </button>
+            <button className="icons-element icons-account">
+              <Link to="/" className="account-btn">
+                <img src={icon2} alt="Account" className={linkIconClassName} />
+              </Link>
+            </button>
+            <button className="icons-element icons-cart Cart-btn">
+              <img src={icon3} alt="Cart" className={linkIconClassName} />
+            </button>
+          </div>
+        </div>
+      </header>
+      <aside className={sidebar ? 'canvas-normal is-open' : 'canvas-normal'}>
```

Comment:
could you create a separate component for this aside?
just too many lines of thml for one component)